### ISSUE TITLE
Vgg Model

### ISSUE THREAD

**Issue by bcarpenter-pub** [tflearn/models#2]
Can you please describe how the weights  for VGG net were trained? Are these the original weights from the paper https://arxiv.org/pdf/1409.1556.pdf or are they a manually trained set of weights.  If so what dataset were they trained on and what accuracy did they achieve.

**Comment by ibadami** [comment]
Can you also show how to read pre-trained weights of google inception v3 model from tensor-flow. They are not saved as .tflearn.

**Comment by KeqiangSun** [comment]
Thanks for your contribution. but there seems to lack the weight named 'conv4_2' and 'fc7/W' when I try to finetune the pretrained VGG16 model. could you please tell me why and what shall i do? Thank you

**Comment by anqiwang0218** [comment]
Thanks for your contribution! But how to get pre-trained vgg model weights and use them in your model  ?? Should I download some other documents or ..?

**Comment by anqiwang0218** [comment]
It's like that your model is just a structure, not a pre-trained model. So how can I read weights into it ?

**Comment by anqiwang0218** [comment]
Or are those weights just inside TFlearn? I am a beginner of Tensorflow and Tflearn. Hope I can get more explanation from you.

**Comment by anqiwang0218** [comment]
@aymericdamien

**Comment by primasanjaya** [comment]
I am asking the same question,

How to load .tflearn pretrained model? is it using model.load() as usual?
